doc/sphinx/azhelpgen/azhelpgen.py

```diff
@@ -35,10 +34,11 @@ def make_rst(self):
 
         for help_file in help_files:
             is_command = isinstance(help_file, _help.CommandHelpFile)
-            yield '.. cli{}:: {}'.format('command' if is_command else 'group', help_file.command)
+            yield '.. cli{}:: {}'.format('command' if is_command else 'group', help_file.command if help_file.command else 'az') #it is top level group az if command is empty
             yield ''
             yield '{}:summary: {}'.format(INDENT, help_file.short_summary)
             yield '{}:description: {}'.format(INDENT, help_file.long_summary)
+            yield '{}:docsource: {}'.format(INDENT, help_file.doc_source)
```

Comment:
I add dedicated dictionary to store the map between group name and the help file path since they are not exactly one-one map, e.g. 'resource' and 'tag' are mapped to same help file under resource module.
The only shortcoming is we must maintain the dictionary if moving the help files.